### ISSUE TITLE
Project not found on signup or login

### ISSUE THREAD

**Issue by SnazzyCoder** [manikandanraji/twitter-clone-backend#8]
A toastified notification saying "Project not found" pops up when signing up or logging in. I assume this is because of crashed backend on heroku..

**Comment by manikandanraji** [comment]
Yep. I kinda broke the backend when trying to fix it. I need to look into it. Meanwhile, I am going to completely rewrite the backend part using nestjs or use some other ORM other than prisma. So it may take sometime. Thanks for reminding.
